pyoptsparse/pyOpt_error.py

```diff
@@ -23,7 +23,11 @@ def __init__(self, message):
                 i += len(word) + 1
         msg += " " * (79 - i) + "|\n" + "+" + "-" * 78 + "+" + "\n"
         print(msg)
-        Exception.__init__(self)
+        self.message = msg
```

Comment:
Do you want to store the formatted message, or the raw `message` string as the attribute? The latter may be easier to work with if you want to parse it.

Reply:
That makes sense, thanks